socorro/processor/mozilla_transform_rules.py

```diff
@@ -411,6 +411,9 @@ def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
 
     def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
         raw_crash['ProductName'] = self.PRODUCT_MAP.get(raw_crash['ProductID'])
+        processor_meta.processor_notes.append(
+            '"Version" missing from esr release raw_crash'
+        )
```

Comment:
I don't understand this change. Best I can figure is that this belongs somewhere else maybe?

Reply:
errant paste. good catch, removing